samples/src/main/trident/NYC-GreenTaxi-Read-Write-Data-To-Kusto.ipynb

```diff
@@ -0,0 +1 @@
0791016","11":"1","12":"N","13":"2","14":"7.5","15":"0.5","16":"0.5","17":"0.3","18":"0.0","19":"0.0","20":"NULL","21":"8.8","22":"1","23":"2015","24":"5","index":6},{"0":"2","1":"2015-05-01 20:00:53","2":"2015-05-01 20:15:06","3":"1","4":"3.18","5":"NULL","6":"NULL","7":"-73.93899536132812","8":"40.805240631103516","9":"-73.97628784179688","10":"40.78804016113281","11":"1","12":"N","13":"1","14":"12.5","15":"1.0","16":"0.5","17":"0.3","18":"2.86","19":"0.0","20":"NULL","21":"17.16","22":"1","23":"2015","24":"5","index":7},{"0":"2","1":"2015-05-01 20:02:24","2":"2015-05-01 20:13:23","3":"1","4":"2.52","5":"NULL","6":"NULL","7":"-73.87102508544922","8":"40.73403549194336","9":"-73.90641784667969","10":"40.72658157348633","11":"1","12":"N","13":"2","14":"11.5","15":"0.5","16":"0.5","17":"0.3","18":"0.0","19":"0.0","20":"NULL","21":"12.8","22":"1","23":"2015","24":"5","index":8},{"0":"2","1":"2015-05-01 20:13:24","2":"2015-05-01 20:15:27","3":"1","4":"0.41","5":"NULL","6":"NULL","7":"-73.92544555664062","8":"40.81804656982422","9":"-73.91799926757812","10":"40.81624984741211","11":"1","12":"N","13":"2","14":"3.5","15":"0.5","16":"0.5","17":"0.3","18":"0.0","19":"0.0","20":"NULL","21":"4.8","22":"1","23":"2015","24":"5","index":9},{"0":"2","1":"2015-05-01 20:11:42","2":"2015-05-01 20:15:21","3":"1","4":"0.63","5":"NULL","6":"NULL","7":"-73.93919372558594","8":"40.80535888671875","9":"-73.93680572509766","10":"40.812416076660156","11":"1","12":"N","13":"1","14":"4.5","15":"0.5","16":"0.5","17":"0.3","18":"1.0","19":"0.0","20":"NULL","21":"6.8","22":"1","23":"2015","24":"5","index":10}],"schema":[{"key":"0","name":"vendorID","type":"int"},{"key":"1","name":"lpepPickupDatetime","type":"timestamp"},{"key":"2","name":"lpepDropoffDatetime","type":"timestamp"},{"key":"3","name":"passengerCount","type":"int"},{"key":"4","name":"tripDistance","type":"double"},{"key":"5","name":"puLocationId","type":"string"},{"key":"6","name":"doLocationId","type":"string"},{"key":"7","name":"pickupLongitude","type":"double"},{"key":"8","name":"pickupLatitude","type":"double"},{"key":"9","name":"dropoffLongitude","type":"double"},{"key":"10","name":"dropoffLatitude","type":"double"},{"key":"11","name":"rateCodeID","type":"int"},{"key":"12","name":"storeAndFwdFlag","type":"string"},{"key":"13","name":"paymentType","type":"int"},{"key":"14","name":"fareAmount","type":"double"},{"key":"15","name":"extra","type":"double"},{"key":"16","name":"mtaTax","type":"double"},{"key":"17","name":"improvementSurcharge","type":"string"},{"key":"18","name":"tipAmount","type":"double"},{"key":"19","name":"tollsAmount","type":"double"},{"key":"20","name":"ehailFee","type":"double"},{"key":"21","name":"totalAmount","type":"double"},{"key":"22","name":"tripType","type":"int"},{"key":"23","name":"puYear","type":"int"},{"key":"24","name":"puMonth","type":"int"}],"truncated":false}},"type":"Synapse.DataFrame"}},"version":"0.1"},"trident":{"lakehouse":{}},"widgets":{}},"nbformat":4,"nbformat_minor":0}
```

Comment:
Change
 The targets where this data has to be written
to
 The target where this data will be written to